Userland/Utilities/gev.cpp

```diff
@@ -0,0 +1,295 @@
+                "    KeyCode: {} (0x{:hex-dump})",
+                (u8)event.key(), (u8)event.key()));
+        } else {
+            outln(String::formatted(
+                "    Key: {}, KeyCode: {} (0x{:hex-dump})",
+                event.text(), (u8)event.key(), (u8)event.key()));
+        }
+
+        outln(String::formatted(
+            "    Modifiers: CTRL: {}, ALT: {}, SHIFT: {}, SUPER: {}",
+            event.ctrl(), event.alt(), event.shift(), event.super()));
+        outln();
+    }
+
+    void print_mouse_event(GUI::MouseEvent& event, String event_type)
+    {
+        if (!(m_all_enabled || m_mouse_enabled))
+            return;
+        outln(event_type);
+        outln(String::formatted("    Position: {}", event.position()));
+        String button = "Unknown";
+        switch (event.button()) {
+        case GUI::MouseButton::None:
+            button = "None";
+            break;
+        case GUI::MouseButton::Left:
+            button = "Left";
+            break;
+        case GUI::MouseButton::Middle:
+            button = "Middle";
+            break;
+        case GUI::MouseButton::Right:
+            button = "Right";
+            break;
+        case GUI::MouseButton::Forward:
+            button = "Forward";
+            break;
+        case GUI::MouseButton::Back:
+            button = "Back";
+            break;
+        default:
+            button = "Unknown";
+        }
+        outln(String::formatted("    Button: {}, Buttons: {}", button, event.buttons()));
+        outln(String::formatted(
+            "    Modifiers: CTRL: {}, ALT: {}, SHIFT: {}, SUPER: {}",
+            event.ctrl(), event.alt(), event.shift(), event.super()));
+        outln(String::formatted("    WheelDelta: {}", event.wheel_delta()));
+        outln();
+    }
+
+    void print_drag_event(GUI::DragEvent& event, String event_type)
+    {
+        if (!(m_all_enabled || m_drag_enabled))
+            return;
+        outln(event_type);
+        outln(String::formatted("    Position: {}", event.position()));
+        outln(String::formatted("    MIMETypes: {}", event.mime_types()));
+        outln();
+    }
+
+    void print_drop_event(GUI::DropEvent& event)
+    {
+        if (!(m_all_enabled || m_drop_enabled))
+            return;
+        outln("Drop:");
+        outln(String::formatted("    Position: {}", event.position()));
+        outln(String::formatted("    Text: {}", event.text()));
+        String mime_data = String::formatted("    MIMETypes: {}", event.mime_data().formats());
+        if (event.mime_data().has_text()) {
+            String data = event.mime_data().text();
+            if (data.length() > 10) {
+                data = data.substring(10);
+            }
+            mime_data = String::formatted("{}\n    MIMEText: {}", mime_data, data);
+        }
+        if (event.mime_data().has_urls()) {
+            mime_data = String::formatted("{}\n    MIMEURLs: {}", mime_data, event.mime_data().urls());
```

Comment:
I suggest using a StringBuilder for this rather than appending `mime_data` to itself in the format string